src/uniprotkb/components/protein-data-views/VisualVariationView.tsx

```diff
@@ -20,96 +24,50 @@ type VariationViewProps = {
-        // eslint-disable-next-line no-param-reassign
-        node.filters = filterConfig;
-      }
-    },
-    [filterElement.defined]
-  );
-
-  const variationElement = useCustomElement(
-    /* istanbul ignore next */
-    () =>
-      import(
-        /* webpackChunkName: "protvista-variation" */ 'protvista-variation'
-      ),
-    'protvista-variation'
-  );
-
-  const protvistaVariationRef = useCallback(
-    (node: {
-      colorConfig: typeof colorConfig;
-      data: VariationViewProps;
-      length: number;
-    }) => {
-      if (node && variationElement.defined && variants) {
+  const navigationRef = useRef<NightingaleNavigation>(null);
+  const setNightingaleVariation = useCallback(
+    (node: NightingaleVariation) => {
+      if (node && sequence && variants) {
         // eslint-disable-next-line no-param-reassign
         node.colorConfig = colorConfig;
         // eslint-disable-next-line no-param-reassign
         node.data = { sequence, variants };
-        // eslint-disable-next-line no-param-reassign
-        node.length = sequence.length;
       }
     },
-    [variationElement.defined, sequence, variants]
-  );
-
-  const navigationElement = useCustomElement(
-    /* istanbul ignore next */
-    () =>
-      import(
-        /* webpackChunkName: "protvista-navigation" */ 'protvista-navigation'
-      ),
-    'protvista-navigation'
-  );
-  const sequenceElement = useCustomElement(
-    /* istanbul ignore next */
-    () =>
-      import(/* webpackChunkName: "protvista-sequence" */ 'protvista-sequence'),
-    'protvista-sequence'
+    [sequence, variants]
   );
 
-  const ceDefined =
-    filterElement.defined &&
-    variationElement.defined &&
-    navigationElement.defined &&
-    sequenceElement.defined;
-
   return (
-    <>
-      {ceDefined && (
-        <div className={styles['variation-view']}>
-          <NightingaleZoomTool length={sequence.length} />
-          <navigationElement.name length={sequence.length} />
-          <sequenceElement.name
-            length={sequence.length}
-            sequence={sequence}
-            height="20"
-            filter-scroll
-            no-scroll
-          />
-          <filterElement.name
-            for="variation-component"
-            ref={protvistaFilterRef}
-          />
-          <variationElement.name
-            id="variation-component"
-            length={sequence.length}
-            ref={protvistaVariationRef}
-            no-scroll
-          />
-        </div>
-      )}
-    </>
+    <div className={styles['variation-view']}>
+      <NightingaleZoomTool
+        length={sequence.length}
+        nightingaleNavigationRef={navigationRef}
+      />
+      <NightingaleNavigationComponent
+        ref={navigationRef}
+        length={sequence.length}
+        height={40}
+      />
+      <NightingaleSequenceComponent
+        length={sequence.length}
+        sequence={sequence}
+        height={20}
+        filter-scroll
```

Comment:
`filter-scroll` behaviour is not captured in the new version - https://github.com/ebi-webcomponents/nightingale/blob/d7b2f1a01554087e840a53465bc2310c2756ce6e/packages/nightingale-new-core/src/mixins/withZoom/index.ts#L92.

What is it supposed to do?